todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksRepository.kt

```diff
@@ -58,6 +59,8 @@ class TasksRepository(
             return
         }
 
+        EspressoIdlingResource.increment() // Set app as busy.
```

Comment:
oh. we're actually modifying production code to test this? :(

Reply:
We always have. No need with mock but you have to in prod.